README.md

```diff
@@ -135,10 +135,14 @@ The `options` object takes the following properties:
 * `options.selfstatus` *(string, optional)*: Own status to display e.g. in email footers, defaults to a standard text.
 * `options.e2ee_enabled` *(boolean, optional)*: Enable E2EE. Defaults to `true`.
 
-#### `dc.continueKeyTransfer(messageId, setupCode)`
+#### `dc.continueKeyTransfer(messageId, setupCode, callback)`
 
 Continue the AutoCrypt key transfer on another device. Corresponds to [`dc_continue_key_transfer()`](https://deltachat.github.io/api/classdc__context__t.html#a5af2cdd80c7286b2a495d56fa6c0832f).
 
+* `messageId` *(int, required)* See deltachat api documentation
```

Comment:
string|integer

Reply:
are you sure about the string?

Reply:
Yes, since you do `Number(messageId)` in the code.

Reply:
oh correct, only looked at the core docu.